client/libs/ga-scroll-depth.js

```diff
@@ -0,0 +1,70 @@
+// Simplified/rewritten from https://github.com/leighmcculloch/gascrolldepth.js
+
+import { onWindowScrollDebounce$ } from '../js/utils/dom-events';
+const startTime = new Date().getTime();
+const getElHeight = el => el.offsetHeight + el.offsetTop;
+
+const getWindowHeight = () => {
+  return window.innerHeight || document.documentElement.clientHeight || document.body.clientHeight;
+};
+
+const getPageYOffset = () => {
+  return window.pageYOffset;
+};
+
+const sendEvent = (distance, timing) => {
```

Comment:
Is there a reason you use `const` over `function`?
I can see the idea of making it immutable a good one.

No hold up, just FMI.